src/content/episodes/13.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Tammikuisia turinoita'
+emoji: ⛄️
+recorded: 2022-01-14
+date: 2022-01-24
+duration: '50:32'
+description: |
+  Clojure!
+  React Native & Expo!
+  NAS-rakentelua!
+  Telegram-botteja!
+  MacBookeja!
+  Ynnä muuta!
+  Haluttiin tehdä "Jouluisia jorinoita" -jakso ennen joulua,
+  mutta eipä tehty,
+  joten tässä vastaava jakso kuukauden myöhässä.
+  Hou hou hou!
+---
+
+## Kuulumisia eli jakson pihvi 👂🥩
+
+- 0:40 Hannes muuttanut isompaan kämppään
+- 2:30 Jokavuotinen [Advent of Code -koodikalenteri][advent-of-code]
+  - [Matiaksen Clojure-ratkaisuja Advent of Codeen][advent-of-clojure]
+  - [Matiaksen Need for Speed -ClojureScript-sovellus][need-for-speed]
+- 6:20 [Hanneksen ostoslistasovellus][mysli] React Nativella + [Expolla][expo]
+- 9:02 NAS-rakentelua tai oikeammin vielä vasta aiheen tutkimista
+- 18:17 Telegram-botteja
+  - [PM2: process manager for Node.js apps][pm2]
+  - [WeShare-mobiilisovellus][weshare]
+  - [Uncanny valley -ilmiö][uncanny-valley],
+    ei kun siis [Dunning{{char.ndash}}Kruger-efekti][dunning-kruger]
+  - [WalkingPad-kävelymatto][walkingpad]
+  - [WalkingPad Pro -pro-kävelymatto][walkingpad-pro]
+- 30:33 Matias vaihtanut työpaikkaa: OP:lta [Loihde Factorille][loihde]
+  - [Jest: JavaScript testing framework][jest]
+- 33:24 Matias vaihtanut ja Hannes vaihtamassa Windowsista Macciin
+
+## Jakson valinnat 🍱
+
+- 41:56 Matias: [Vinland Saga -manga][vinland-saga]
+- 44:56 Hannes: [Silicon Valley -TV-sarja][silicon-valley]
+- 47:03 Hannes: [Logitech MX ERGO -pallohiiri][mx-ergo]
+
+[advent-of-clojure]: https://github.com/mtsknn/advent-of-clojure
+[advent-of-code]: https://adventofcode.com/
+[dunning-kruger]: https://www.google.com/search?q=dunning-kruger+effect&tbm=isch
+[expo]: https://expo.dev/
+[jest]: https://jestjs.io/
+[loihde]: https://loihdefactor.com/
+[mx-ergo]: https://www.logitech.com/fi-fi/products/mice/mx-ergo-wireless-trackball-mouse.910-005179.html
+[mysli]: https://github.com/Steellow/mysli
+[need-for-speed]: https://github.com/mtsknn/need-for-speed
+[pm2]: https://github.com/Unitech/pm2
+[silicon-valley]: https://www.imdb.com/title/tt2575988/
+[uncanny-valley]: https://fi.wikipedia.org/wiki/Outo_laakso
+[vinland-saga]: https://www.mangaupdates.com/series.html?id=1568
```

Comment:
@Steellow Can you think of a better url for Vinland Saga? 🤔

Reply:
Either MAL or AL link would be great, I prefer AL

https://anilist.co/manga/30642/Vinland-Saga/